tracing/README.md

```diff
@@ -69,13 +70,17 @@ _n_ new items were taken from a queue, and so on.
 to indicate that some code takes place within the context of that `Span`:
 
 ```rust
+# #[macro_use] extern crate tracing;
+# use tracing::Level;
+# fn main() {
```

Comment:
I think if we no longer need `extern crate`, we don't need to explicitly have a main function? IIRC the issue is generally that `extern crate` cannot be inside a function?

Reply:
Yea, thats the main blocker I believe. with 2018 this should be doable without skeptic